app/services/report.py

```diff
@@ -62,3 +60,36 @@ def add_report(self, twitter_id: str, reporter_id: str) -> Report:
 
         report_db.save()
         return report_db
+
+    def approve_report(self, twitter_id: str):
+        report_db = Report.objects(
+            report_key__twitter_id=twitter_id, expired=False
+        ).first()
+        if report_db:
+            report_db.update(expired=True)
+
+            full_details = self.twitter_scraper.get_full_details(
+                twitter_id, tweets_num=2
+            )
+            label = 0 if report_db.score >= 0.5 else 1
+            ProcessedReport.objects(twitter_id=twitter_id).update_one(
+                user=full_details._json, label=label, upsert=True
+            )
+
+    def reject_report(self, twitter_id: str):
+        Report.objects(report_key__twitter_id=twitter_id, expired=False).first().update(
+            expired=True
+        )
+
+    def export(self) -> List[ProcessedReportResponse]:
+        processed_report_list = []
+        for report in ProcessedReport.objects:
+            report_dict = report.to_mongo()
```

Comment:
@dirtyg2120 Why?

Reply:
the return type of `report.user` in that `for` loop is `mongoengine.base.datastructures.BaseDict`, so i just want to change it to primitive `dict`.